godtools/App/Services/Renderer/Renderer/Multiplatform/MobileContentMultiplatformRenderer.swift

```diff
@@ -67,15 +67,15 @@ class MobileContentMultiplatformRenderer: MobileContentRendererType {
             rendererState: rendererState
         )
         
-        if let renderableView = recurseAndRender(renderableModel: pageModel, renderableModelParent: nil, rendererPageModel: rendererPageModel, containerModel: nil) {
+        if let renderableView = recurseAndRender(renderableModel: pageModel, renderableModelParent: nil, rendererPageModel: rendererPageModel, containerModel: nil, primaryLanguage: primaryRendererLanguage) {
             return .success(renderableView)
         }
                 
         let failedToRenderPageError: Error = NSError(domain: "", code: -1, userInfo: [NSLocalizedDescriptionKey: "Failed to render page."])
         return .failure(failedToRenderPageError)
     }
     
-    private func recurseAndRender(renderableModel: MobileContentRenderableModel, renderableModelParent: MobileContentRenderableModel?, rendererPageModel: MobileContentRendererPageModel, containerModel: MobileContentRenderableModelContainer?) -> MobileContentView? {
+    private func recurseAndRender(renderableModel: MobileContentRenderableModel, renderableModelParent: MobileContentRenderableModel?, rendererPageModel: MobileContentRendererPageModel, containerModel: MobileContentRenderableModelContainer?, primaryLanguage: LanguageModel) -> MobileContentView? {
```

Comment:
Don't need to pass primary language here as it is available in MobileContentRenderableModel.